policy/modules/services/tpm2.if

```diff
@@ -56,3 +145,23 @@ interface(`tpm2_status_abrmd',`
 
 	allow $1 tpm2_abrmd_unit_t:service status;
 ')
+
+########################################
+## <summary>
+##	access tpm2-abrmd fifos
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access
+##	</summary>
+## </param>
+#
+interface(`tpm2_rw_pipes_abrmd',`
```

Comment:
`tpm2_rw_abrmd_pipes`

Reply:
Updated - sorry for the delay - I was testing with some other changes and everything seems OK.